api-v2/src/student/entities/student.entity.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  Entity,
+  Column,
+  PrimaryGeneratedColumn,
+  BeforeInsert,
+  CreateDateColumn,
+  UpdateDateColumn,
+  OneToMany,
+} from 'typeorm';
+import * as bcrypt from 'bcrypt';
+import { Exclude } from 'class-transformer';
+import { ScheduleCourse } from '../../../../common/types';
+import { Plan } from '../../plan/entities/plan.entity';
+
+@Entity()
```

Comment:
it's confusing what types are db types and what are not. what KOH did was put the DTO types in `common` package (shared between frontend and backend) and then put db entities in .entity.ts files (in the relevant module folder in backend), see https://github.com/sandboxnu/office-hours/blob/master/packages/server/src/course/course.entity.ts

Reply:
So I agree with moving DTO types into the common package after yarn workspace move! 

But I didn't quite understand why it was difficult to see which types were entity and which weren't since entity files are named `x.entity.ts` in the `entity` directory and DTOs are `x.dto.ts` in the `dto` directory. 

Did you mean that we should move the `.entity.ts` files out of the `entity` directory and into the module directory?

Reply:
Ohhb LOL ignore me haha i didnt see file extension